codecov.yml

```diff
@@ -78,16 +78,17 @@ comment:
   # This is the addition of carry forward builds and fresh builds, thus, it's the addition
   # of the FE and BE builds
   after_n_builds: 22
-  layout: 'diff, files'
-  # Update, if comment exists. Otherwise post new.
-  behavior: default
-  # Comments will only post when coverage changes. Furthermore, if a comment
-  # already exists, and a newer commit results in no coverage change for the
-  # entire pull, the comment will be deleted.
-  require_changes: true
+  # Show only the git diff / patch coverage, and exclude any comparison between base and head:
+  layout: 'condensed_header, condensed_files, condensed_footer'
+  hide_project_coverage: true
+  # Update, if exists. Otherwise post new. Skip if deleted.
+  behavior: once
+  # Comments will only post when coverage drops OR there are uncovered lines of code
+  require_changes: 'coverage_drop OR uncovered_patch'
   require_base: true # must have a base report to post
   require_head: true # must have a head report to post
   require_bundle_changes: True # only post a comment when there are bundle size changes
+  bundle_change_threshold: '10Kb'
```

Comment:
I made a very small PR and it increased 7.19kb. This should be a good starting point to only report when unwanted increases happen.
<img width="492" alt="image" src="https://github.com/user-attachments/assets/0bf63b5f-a1ef-408b-998b-e4d1b9da43a6">